data/management/commands/makeusertoken.py

```diff
@@ -0,0 +1,36 @@
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import User
+from rest_framework.authtoken.models import Token
+
+
+class Command(BaseCommand):
+    help = 'Adds user with specified token name'
+
+    def add_arguments(self, parser):
+        parser.add_argument('username')
+        parser.add_argument('password')
+        parser.add_argument('tokenkey')
+
+    def _create_user(self, username, password):
+        user, created = User.objects.get_or_create(username=username, is_staff=True, is_superuser=True)
+        if created:
+            self.stdout.write("User '{}' created ith id {}".format(username, user.id))
+        else:
+            self.stderr.write("User '{}', already exists with id {}".format(username, user.id))
+        user.set_password(password)
+        user.save()
+        return user
+
+    def _create_token(self, user, tokenkey):
+        token, created = Token.objects.get_or_create(key=tokenkey, user=user)
```

Comment:
Token has a unique constraint, so this would fail if the user already has a token and a different token is specified.